Aspose.Slides.WebExtensions/Templates/common/shape.html

```diff
@@ -4,11 +4,12 @@
     Shape contextObject = Model.Object;
     
     var origin = Model.Local.Get<Point>("origin");
-    var positionStyle = string.Format("left: {0}px; top: {1}px; width: {2}px; height: {3}px;",
+    var positionStyle = ShapeHelper.GetPositionStyle(contextObject, origin);
+    /*var positionStyle = string.Format("left: {0}px; top: {1}px; width: {2}px; height: {3}px;",
```

Comment:
@polonair ,

This commented code is needed for something? Can we delete it?

Reply:
Yes< we can delete it